modules/admin/src/main/resources/webapp/js/app.js

```diff
@@ -108,7 +108,7 @@
                }
             })
             .state('admin.browse', {
-               url: '/browse',
+               url: '/browse?queueName',
```

Comment:
@sebbrudzinski 
1. Yes, in my MOTECH I had two queues without messages, and after click on queue name, list was empty, and third with something about 50 messages. Everything seems to work.

2. In topics doesn't exists possibility of displaying details.